src/OpenTelemetry/.publicApi/netstandard2.0/PublicAPI.Unshipped.txt

```diff
@@ -1,8 +1,145 @@
+OpenTelemetry.Metrics.Aggregation.LastValue = 3 -> OpenTelemetry.Metrics.Aggregation
+OpenTelemetry.Metrics.Aggregation.Sum = 2 -> OpenTelemetry.Metrics.Aggregation
+OpenTelemetry.Metrics.AggregationTemporality
+OpenTelemetry.Metrics.AggregationTemporality.Cumulative = 1 -> OpenTelemetry.Metrics.AggregationTemporality
+OpenTelemetry.Metrics.AggregationTemporality.Delta = 2 -> OpenTelemetry.Metrics.AggregationTemporality
+OpenTelemetry.Metrics.AggregationTemporalityAttribute
+OpenTelemetry.Metrics.AggregationTemporalityAttribute.AggregationTemporalityAttribute(OpenTelemetry.Metrics.AggregationTemporality supported) -> void
+OpenTelemetry.Metrics.AggregationTemporalityAttribute.AggregationTemporalityAttribute(OpenTelemetry.Metrics.AggregationTemporality supported, OpenTelemetry.Metrics.AggregationTemporality preferred) -> void
+OpenTelemetry.Metrics.AggregationTemporalityAttribute.Preferred.get -> OpenTelemetry.Metrics.AggregationTemporality
+OpenTelemetry.Metrics.AggregationTemporalityAttribute.Supported.get -> OpenTelemetry.Metrics.AggregationTemporality
+OpenTelemetry.Metrics.BaseExportingMetricReader
+OpenTelemetry.Metrics.BaseExportingMetricReader.BaseExportingMetricReader(OpenTelemetry.BaseExporter<OpenTelemetry.Metrics.Metric> exporter) -> void
+OpenTelemetry.Metrics.BaseExportingMetricReader.SupportedExportModes.get -> OpenTelemetry.Metrics.ExportModes
+OpenTelemetry.Metrics.BatchMetricPoint
+OpenTelemetry.Metrics.BatchMetricPoint.BatchMetricPoint() -> void
+OpenTelemetry.Metrics.BatchMetricPoint.Dispose() -> void
+OpenTelemetry.Metrics.BatchMetricPoint.Enumerator
+OpenTelemetry.Metrics.BatchMetricPoint.Enumerator.Current.get -> OpenTelemetry.Metrics.MetricPoint
+OpenTelemetry.Metrics.BatchMetricPoint.Enumerator.Dispose() -> void
+OpenTelemetry.Metrics.BatchMetricPoint.Enumerator.Enumerator() -> void
+OpenTelemetry.Metrics.BatchMetricPoint.Enumerator.MoveNext() -> bool
+OpenTelemetry.Metrics.BatchMetricPoint.Enumerator.Reset() -> void
+OpenTelemetry.Metrics.BatchMetricPoint.GetEnumerator() -> OpenTelemetry.Metrics.BatchMetricPoint.Enumerator
+OpenTelemetry.Metrics.DataValue
+OpenTelemetry.Metrics.DataValue.DataValue() -> void
+OpenTelemetry.Metrics.DataValue.Value.get -> object
+OpenTelemetry.Metrics.ExportModes
+OpenTelemetry.Metrics.ExportModes.Pull = 2 -> OpenTelemetry.Metrics.ExportModes
+OpenTelemetry.Metrics.ExportModes.Push = 1 -> OpenTelemetry.Metrics.ExportModes
+OpenTelemetry.Metrics.ExportModesAttribute
+OpenTelemetry.Metrics.ExportModesAttribute.ExportModesAttribute(OpenTelemetry.Metrics.ExportModes supported) -> void
+OpenTelemetry.Metrics.ExportModesAttribute.Supported.get -> OpenTelemetry.Metrics.ExportModes
+OpenTelemetry.Metrics.HistogramConfiguration
+OpenTelemetry.Metrics.HistogramConfiguration.BucketBounds.get -> double[]
+OpenTelemetry.Metrics.HistogramConfiguration.BucketBounds.set -> void
+OpenTelemetry.Metrics.HistogramConfiguration.HistogramConfiguration() -> void
+OpenTelemetry.Metrics.IDataPoint
```

Comment:
could be cleaned up now.
(will be a separate PR along with reviewing entire public API)